frontend/src/components/Utils/time.js

```diff
@@ -61,8 +61,11 @@ export function timestampToFormatted(msTimestamp, timezone = "America/New_York")
 /**
  * Return a Firestore Timestamp corresponding to the date in `dateStr`.
  *
+ * TODO(Issue #89): Remove this function when integrating timezones for the
+ * ViewTrips components.
+ *
  * @param {string} dateStr String containing a date in the form 'YYYY-MM-DD'.
- * @return {firebase.firestore.Timestamp} Firestore timestamp object created.
+ * @return {firebase.firestore.Timestamp} Firestore timestamp object.
  */
 export function getTimestampFromDateString(dateStr) {
   const dateParts = dateStr.split('-').map(str => +str);
```

Comment:
Fixed.

Reply:
Scratch that. So that worked, however when passing the ISO string the timezone defaulted to UTC and that is undesirable because the other methods use the user's current default timezone. So I kept this the same for now until fixing #89 so that all date operations use the current user's timezone.